code/game/objects/items/storage/lockbox.dm

```diff
@@ -122,7 +122,7 @@
 			return TRUE
 
 /obj/item/storage/lockbox/medal/PopulateContents()
```

Comment:
`/obj/item/storage/lockbox/medal/PopulateContents()
	new /obj/item/clothing/accessory/medal/silver/valor(src)
	new /obj/item/clothing/accessory/medal/silver/security(src)
	new /obj/item/clothing/accessory/medal/bronze_heart(src)
	for(var/i in 1 to 3)
		new /obj/item/clothing/accessory/medal/conduct(src)
	for(var/i in 1 to 3
	    new /obj/item/clothing/accessory/medal/bronze_heart(src)`

I think this would be more appropriate for the medals within